src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_DefaultUserProfileScreen.tsx

```diff
@@ -27,6 +27,10 @@ export const Profile_DefaultUserProfileScreen = () => {
       (userProfile) => userProfile.userTypeString === defaultUserTypeString,
     ) ?? userProfiles[0];
 
+  const userProfilesWithoutHiddenCategories = userProfiles.filter(
```

Comment:
Nit-picky, but not quite sure what "hidden categories" is. Maybe we could use something like this?
```suggestion
  const defaultTravellerSelectionUserProfiles = userProfiles.filter(
```

Reply:
What about just `selectableUserProfiles`?

Reply:
Yep 😊